modules/steam.nix

```diff
@@ -259,6 +259,18 @@ in
       services.logind.extraConfig = ''
         HandlePowerKey=ignore
       '';
+
+      # HACK: This is a temporary workaround to allow Steam to suspend the
+      # system while running inside a transient slice.
```

Comment:
Is shutdown/reboot affected by the same issue?

Reply:
Right, I haven't tried them through the UI in a long time :sweat_smile: Yes, they are affected and I have added them into the rule.